### DISCUSSION TITLE
Apple Pro Display XDR

### DISCUSSION THREAD

**Discussion by zolomatok** [emoacht/Monitorian#310] [Compatibility]
Not really an issue, but a question. I'm sorry, I don't know where else to ask.

Does Monitorian work with the XDR on a windows PC? (with a 30 series GeForce card if that makes a difference) (Not a mac that's running bootcamp)

Thank you!

**Comment by emoacht** [comment]
It depends on whether the mac and the display support DDC/CI and I have no idea on it.
I have confirmed this app works on intel MacBook and Dell monitor but I have never tested on Apple Silicon mac with Apple display.

**Reply by zolomatok** [reply]
Thanks for the quick reply!
What I'm curious about is whether it works on stock Windows with a Windows PC, with a  GeForce 30XX card.

I know AMD has a utility that can adjust the XDR's brightness in Windows, but I'm not sure if it's possible with an nVidia card.

**Reply by emoacht** [reply]
I haven't heard any information on DDC/CI capability of that display. Perhaps it lacks this capability.
